api-gateway/src/main/resources/static/scripts/inventory-product-list/inventory-product-list.template.html

```diff
@@ -53,11 +53,11 @@ <h2>Inventory Products</h2>
     </thead>
     <tr ng-repeat="product in $ctrl.inventoryProductList | filter:$ctrl.query track by product.productId">
         <td><span>{{product.productName}}</span></td>
-            <td><span>{{product.productId}}</span></td>
+        <td><span>{{product.productId}}</span></td>
         <td><span>{{product.productQuantity}}</span></td>
-        <td><span>{{product.productPrice}}</span></td>
+        <td><span>$ {{product.productPrice}}</span></td>
```

Comment:
[Nitpick]: Do you think the format should be more like "$20.99" instead of "$ 20.99"?

Reply:
Can't the other way be said about it being 20.99$ or 20.99 $....

Reply:
I've changed it to be 20.99$